src/actions/__tests__/analytics.ts

```diff
@@ -50,6 +53,12 @@ jest.mock('react-native-omniture', () => ({
 }));
 jest.mock('../../auth/authUtilities');
 
+((FBAnalytics as unknown) as jest.Mock).mockReturnValue({
+  logEvent: jest.fn(),
+  logScreenView: jest.fn(),
+  setUserProperties: jest.fn(),
+});
```

Comment:
I don't think you need this anymore...

Reply:
Actually, if I remove this the tests fail for this file.

Reply:
Probably because they were written for this mock instead of the global mock?